charts/lagoon-logging/values.yaml

```diff
@@ -233,9 +233,12 @@ excludeNamespaces:
 # This may require tweaking to handle high volumes of logs.
 clusterOutputBuffer:
   flush_thread_count: 32
-  timekey: 1m
-  timekey_wait: 10s
+  timekey: 10s # create chunks every 10 secs, therefore possible to flush every 10sec
```

Comment:
Due to the volume of logs coming from the cluster I actually ran into issues with setting the timekey too low. I was getting errors like this due to running out of file descriptors due to lots of little chunks:
```
<Fluent::Config::Section {"@label":"@d18536be251369c8ce5b2628a6f329f2","tag":"","matches":["<Fluent::Config::Section {\"labels\":{},\"namespaces\":[],\"hosts\":[],\"container_names\":[],\"negate\":false}>"]}>
#<Thread:0x000056346293a188@event_loop@/usr/lib/ruby/gems/2.6.0/gems/fluentd-1.9.2/lib/fluent/plugin_helper/thread.rb:70 run> terminated with exception (report_on_exception is true):
/usr/lib/ruby/2.6.0/socket.rb:1313:in `__accept_nonblock': No file descriptors available - accept(2) (Errno::EMFILE)
	from /usr/lib/ruby/2.6.0/socket.rb:1313:in `accept_nonblock'
	from /usr/lib/ruby/gems/2.6.0/gems/cool.io-1.6.0/lib/cool.io/listener.rb:46:in `on_readable'
	from /usr/lib/ruby/gems/2.6.0/gems/cool.io-1.6.0/lib/cool.io/loop.rb:88:in `run_once'
	from /usr/lib/ruby/gems/2.6.0/gems/cool.io-1.6.0/lib/cool.io/loop.rb:88:in `run'
	from /usr/lib/ruby/gems/2.6.0/gems/fluentd-1.9.2/lib/fluent/plugin_helper/event_loop.rb:93:in `block in start'
	from /usr/lib/ruby/gems/2.6.0/gems/fluentd-1.9.2/lib/fluent/plugin_helper/thread.rb:78:in `block in thread_create'
```
Have you seen any errors like that in the `logging-operator` fluentd?

Reply:
Checked the cluster and don't see any issues. If there are problems we can always raise this again :+1: